code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -5,9 +5,9 @@
 	icon_state = "cypherkey"
 	item_state = ""
 	w_class = W_CLASS_TINY
+	origin_tech = Tc_MAGNETS + "=4;" + Tc_ENGINEERING + "=4;" + Tc_MATERIALS + "=4;" + Tc_PROGRAMMING + "=4"
```

Comment:
Is this the new RnD speedrun?

Reply:
I honestly didn't have that in mind ! Thanks for the insight.